CONTRIBUTING.md

```diff
@@ -1,65 +1,41 @@
 
-If no existing type matches the terms you intend to track, check the open discussions in the category “New Type” in the [Discussions section](https://github.com/opentermsarchive/terms-types/discussions) of this repository.
-
-If no discussion seems to match the type you are interested in adding, [start a new “New Type” discussion](https://github.com/OpenTermsArchive/terms-types/discussions/new?category=new-type) and call it “Add _your proposed terms type name_”.
-
-### Define the type name
-
-Propose a unique and descriptive name for the new terms type.
-
-It should be the most commonly used and most internationally understandable for this type. It doesn't have to, and probably should not, match the service-provided name. Make use of tools such as Google Trends, Linguee and thesaurus to find the name that strikes the best balance between generality, precision and multilingualism.
-
-> For example, the “Imprint” type is [called this way](https://github.com/OpenTermsArchive/engine/issues/801) as a balance between the French “mentions légales”, the German “impressum”, and other considerations.
-
-Ensure that this name (or any close one) is not already used in the [database](./termsTypes.json).
-
-Note that service-specific types (such as “Twitter Privacy Policy”) are not allowed. Terms types aim at allowing comparisons across services and should thus be generic.
-
-### Provide examples
-
-Provide at least the terms you intend to track as example of the new terms type, with relevant information on its source and context:
-
-- Include a direct link to the source document.
-- Name the service provider and describe its service.
-- Provider-given terms name: provide the title or name given to these terms by the service provider.
-
-### Add references
-
-Include any relevant [references](./README.md#references) that may help in understanding the purpose of this type. This could include legal definitions, discussions, or any other resource that provides context or background information.
-
-References will then be listed in the type definition, with their URL and a title describing the content and source.
+First of all, double-check that it does not already exist in the [database](./termsTypes.json). If it does, use the associated type for tracking. 
 
-### Build consensus
+Then check the [open discussions](https://github.com/opentermsarchive/terms-types/discussions) for this repository to make sure that there is not already a discussion about the type you intend to add.
 
-Publicise your suggestion across Open Terms Archive channels and engage the community. Throughout the discussion, the name and the triptych should be challenged, taking into account international perspectives on phrasings and the variety of definitions across jurisdictions.
+If no discussion seems to match the type you are interested in adding, [start a new discussion](https://github.com/OpenTermsArchive/terms-types/discussions/new?category=submitted) by filling in this form.
```

Comment:
“this form” is ambiguous since it doesn't have a link 🙂 

```suggestion
If no discussion seems to match the type you are interested in adding, [start a new discussion](https://github.com/OpenTermsArchive/terms-types/discussions/new?category=submitted).
```